src/schema.d.ts

```diff
@@ -62,7 +63,7 @@ export function accumulateErrors<T>(
     schema: Schema<T>,
 ): Error<T> | undefined;
 
-export function accumulateDifferentialErrors<T>(
+export function accumulateDifferentialErrors<T, V>(
```

Comment:
Why add a generic type that is not used?

Reply:
Yes